src/Enforce/runnables/rules/helpers/extend_negative_rules/index.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+/**
+ * Collects rules with `negativeForm` attributes.
+ * When found, it creates a new function that flips
+ * the value of the original rule.
+ * @param {Object} rules - enforce rules object
+ * @returns {Object} extended rules object
+ */
+function extendNegativeRules(rules: EnforceRules) {
+
+    for (const rule: string in rules) {
+        const negativeForm: string = rules[rule].negativeForm;
+
+        if (negativeForm) {
+            rules[negativeForm] = (...args) => !rules[rule](...args);
```

Comment:
can you explain this line, I don't understand the `...`

Reply:
I use it to pass on all the arguments forward to the actual rule.
![image](https://user-images.githubusercontent.com/11255103/37251148-c50aa19a-2512-11e8-9917-3c175d315a1e.png)